canonical-web-and-design/microk8s.io
# Update to Vanilla 3.0

## Done

- Update Sass
- Update Vanilla version and import
- Deleted `pattern_tiles` and other scss classes that weren't being used
- Updated variables to new names
- Fixed broken nav 
- Added `u-hide--medium` class to the video on the homepage to fix it filling the whole width of the screen on medium screen sizes

## QA

- View site at: 
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the site for anything breaking. Check particularly the nav and everything at different screen sizes
